src/sorting/README.md

```diff
@@ -180,8 +180,14 @@ __Properties__
 From [Wikipedia][tim-wiki]:  Timsort is a hybrid stable sorting algorithm, derived from merge sort and insertion sort, designed to perform well on many kinds of real-world data. It was implemented by Tim Peters in 2002 for use in the Python programming language. The algorithm finds subsequences of the data that are already ordered (runs) and uses them to sort the remainder more efficiently. This is done by merging runs until certain criteria are fulfilled. Timsort has been Python's standard sorting algorithm since version 2.3. It is also used to sort arrays of non-primitive type in Java SE 7, on the Android platform, in GNU Octave, on V8, Swift, and Rust.
 
 __Properties__
-* Worst-case performance O(n log n)
-* Best-case performance	O(n)
+* Worst-case performance O(max element size(ms))
+* Best-case performance	O(max element size(ms))
+
+### [Sleep](./sleep_sort.rs)
+![alt text][sleep-image]
+
+From [Wikipedia][bucket-sort-wiki]: This is an idea that was originally posted on the message board 4chan, replacing the bucket in bucket sort with time instead of memory space.
```

Comment:
Since we could not confirm wikipedia in English, I refer to the section described in Japanese.
(If necessary, I would be willing to translate it into English and create a wikipedia page.)

Reply:
Having a page in English would be nice. I can merge it with Japanese reference, and then you translate it and make a PR with updated English reference when you have time and energy to do so.